test/powershell/engine/Module/testmodulerunspace/ModuleWithDependencies2/2.0/ModuleWithDependencies2.psd1

```diff
@@ -0,0 +1,129 @@
+#
+# Module manifest for module 'ModuleWithDependencies2'
+#
+# Generated by: manikb
+#
+# Generated on: 10/29/2015
+#
```

Comment:
Can you please remove all comments in this file? They make it hard to see what fields are actually used.